signer/local_cosigner.go

```diff
@@ -114,14 +117,15 @@ func (cosigner *LocalCosigner) SaveLastSignedState(signState SignStateConsensus)
 
 func NewLocalCosigner(cfg LocalCosignerConfig) *LocalCosigner {
 	cosigner := &LocalCosigner{
-		key:           cfg.CosignerKey,
-		lastSignState: cfg.SignState,
-		rsaKey:        cfg.RsaKey,
-		hrsMeta:       make(map[HRSTKey]HrsMetadata),
-		peers:         make(map[int]CosignerPeer),
-		total:         cfg.Total,
-		threshold:     cfg.Threshold,
-		address:       cfg.Address,
+		key:                                     cfg.CosignerKey,
+		lastSignState:                           cfg.SignState,
+		rsaKey:                                  cfg.RsaKey,
+		hrsMeta:                                 make(map[HRSTKey]HrsMetadata),
```

Comment:
this and other references to `hrsMeta` can probably move into the struct for the "local" threshold ed25519 signature implementation.